docs/tutorials/migrations/nvm.md

```diff
@@ -0,0 +1,122 @@
+* **You need to manage versions of other JavaScript runtimes, like Bun or Deno.** While nvm is, as the name denotes, used for managing Node.js versions, that's all it does. You can use Flox to install node, but you can also use it to install different JavaScript runtimes, including [Bun](https://bun.sh/){:target="\_blank"} and [Deno](https://deno.com/){:target="\_blank"}.
+
+## Install Flox
+Download and install Flox as described in our [installation instructions][install_flox]{:target="\_blank"}.
+
+## Create a Flox environment in your existing project and install Node.js
+Run `flox init` from the relevant project directory (see [creating environments][creating_tutorial]{:target="\_blank"} for details), and then search for the version of Node.js that is currently included in your `.nvmrc`, or in the `"engines"` property of your `package.json`. (Note that node is listed as `nodejs` and variants thereof in the Flox catalog.)
+
+```zsh
+➜  node-project git:(main) ✗ flox search nodejs
+nodejs       Event-driven I/O framework for the V8 JavaScript engine
+nodejs_14    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_16    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_18    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_19    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_20    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_21    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_22    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_23    Event-driven I/O framework for the V8 JavaScript engine
+nodejs-10_x  Event-driven I/O framework for the V8 JavaScript engine
+
+Showing 10 of 67 results. Use `flox search nodejs --all` to see the full list.
+
+Use 'flox show <package>' to see available versions
+```
+
+There is _a lot_ of software in the Flox catalog, so you should have no trouble finding the Node.js version you need. For this project, perhaps you initially decide to install Node.js v20.18.1, "Iron" (LTS).
+
+```zsh
+➜  node-project git:(main) ✗ flox install nodejs
+✅ 'nodejs' installed to environment 'node-project'
+```
+## Verify the Node.js version
+Now that you've installed Node.js, you activate the Flox environment to verify that it has the the version you expect.
+
+```zsh
+➜  node-project git:(main) ✗ flox activate
+✅ You are now using the environment 'node-project'.
+To stop using this environment, type 'exit'
+
+...
+flox [node-project] ➜  node-project git:(main) ✗ node -v
+v20.18.1
+```
+
+## Update the Node.js version
+If you want to install a different node version, you can edit your environment to include the version you need.
```

Comment:
```suggestion
If you want to install a different Node version, you can edit your environment to include the version you need.
```

Reply:
When referring to node without using the full name of Node.js, I would keep it lowercase. I don't think any JS dev would capitalize the "n" without also appending the extension. If it's considered too informal for the docs to have the more colloquial "node" in this migration guide, I'll just change them all to "Node.js." However, if a little informality is ok, I would like to throw "node" in there a few times as a mark of authenticity (i.e., that this was written by someone familiar with JavaScript and the associated terminology).